gp/hl_api.py

```diff
@@ -1,18 +1,21 @@
 import numpy as np
+from typing import Optional, Callable
+from .abstract_individual import AbstractIndividual
+from .abstract_population import AbstractPopulation
+from .ea import MuPlusLambda
 
 
 def evolve(
-    pop,
-    objective,
-    ea,
-    max_generations,
-    min_fitness,
-    print_progress=False,
-    *,
-    callback=None,
-    label=None,
-    n_processes=1,
-):
+    pop: AbstractPopulation,
+    objective: Callable[[AbstractIndividual], AbstractIndividual],
```

Comment:
shouldn't the objective just take one individual?

Reply:
This defines the arguments and return value: https://docs.python.org/3/library/typing.html#typing.Callable

Reply:
thanks!